### ISSUE TITLE
Can't focus on Android

### ISSUE THREAD

**Issue by ethanyuwang** [wix/react-native-zss-rich-text-editor#108]
I initialized a fresh react native project ("react-native": "0.55.4",) and copied the example code. However on Android `focusTitle` and `focusContent` both don't work 

---------------------------Update--------------------------------

`focusContent` would not work initially. However, after I actually tap the editor to focus and then programmatically blur it, `focusContent` would work as expected.

**Comment by rasselll** [comment]
if your targetSDKVersion is greater than 24, seems like it wont work

**Comment by cherniv** [comment]
Doesnt work for me even with targetSDKVersion  = 24

**Comment by jeffie** [comment]
have a solution now?

**Comment by ethanyuwang** [comment]
@rasselll  I have targetSdkVersion = 22 and it doesn't work

**Comment by ethanyuwang** [comment]
Any updates?

**Comment by saurabhabh** [comment]
Any Updates?

**Comment by saurabhabh** [comment]
Just a quick update I noticed that because of my other styles this was not working properly. I made changes to them and it worked. Hint: Check your styling.

**Comment by ethanyuwang** [comment]
@saurabhabh what styling changes did you make?

**Comment by saurabhabh** [comment]
https://github.com/wix/react-native-zss-rich-text-editor/issues/138#issuecomment-441521534 - @ethanyuwang  Check this comment. Check your flex direction. It only works for me if the parent container's flex direction is set to column.

**Comment by eladgel** [comment]
Here is our solution: @ethanyuwang 

- Change this in your package.json  "react-native-zss-rich-text-editor": "github:honeybook/react-native-zss-rich-text-editor.git#master",
- Create a native module that does this 
![image](https://user-images.githubusercontent.com/412137/51212240-9d498600-1920-11e9-8bf9-e52f6281ad8c.png)
- Add this property to the RTE 
![image](https://user-images.githubusercontent.com/412137/51212389-0d580c00-1921-11e9-8af4-e923ba31b441.png)
- Call these 2 on Editor Initialisation 
![image](https://user-images.githubusercontent.com/412137/51212569-80fa1900-1921-11e9-9de0-21f461130f7f.png)

#win

**Comment by ethanyuwang** [comment]
@eladgel  Can I ask you what changes you made in your fork?

**Comment by eladgel** [comment]
@ethanyuwang just go to the fork and view the changed files

**Comment by ethanyuwang** [comment]
@eladgel  Thanks a lot I have made mine working, but it seems like you only need to change `WebViewBridgeManager.java`  in `react-native-webview-bridge` though? This is what I did:

1. Add new prop `requestFocus` in `react-native-webview-bridge`:
<img width="993" alt="screen shot 2019-01-22 at 4 58 07 pm" src="https://user-images.githubusercontent.com/17556945/51575430-ea6cb100-1e66-11e9-9625-7cf9053e487e.png">

2. Add the prop in where you render `RichTextEditor` as you mentioned:
<img width="313" alt="screen shot 2019-01-22 at 4 59 38 pm" src="https://user-images.githubusercontent.com/17556945/51575490-21db5d80-1e67-11e9-8a98-91729fa7363b.png">
